rwengine/src/ai/AIGraph.cpp

```diff
@@ -112,14 +113,8 @@ void AIGraph::gatherExternalNodesNear(const glm::vec3& center,
                 continue;
             }
             auto& external = gridNodes[i];
-            for (AIGraphNode* node : external) {
-                if (glm::distance2(center, node->position) < radius * radius) {
-                    if (node->type == type)
-                    {
-                        nodes.push_back(node);
-                    }
-                }
-            }
+            copy_if(external.begin(), external.end(), back_inserter(nodes),
+                    [&center, &radius, &type](const AIGraphNode* node) { return node->type == type && glm::distance2(center, node->position) < radius * radius; });
```

Comment:
This line is too long (>80 columns). The ``.clang-format`` file in the repository specifies the code style that should be used, and can be applied automatically using clang format.